web-ui/src/pages/WebPortal.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import "./WebPortal.css";
+import RightSidebar from "../components/onboarding_sidebar/RightSidebar";
+import CultureVideoPage from "./CultureVideoPage";
+import BackgroundInformationPage from "./BackgroundInformationPage";
+import IntroductionSurveyPage from "./IntroductionSurveyPage";
+import WorkingLocationPage from "./WorkingLocationPage";
+import EquipmentPage from "./EquipmentPage";
+import DocumentSigningPage from "./DocumentSigningPage";
+import Congratulations from "./CongratulationsPage";
+import { Container } from "@mui/material";
+
+function WebPortal() {
+  const menuList = [
+    {
+      index: 0,
+      name: "Culture Video",
+      title: "Lorem ipsum",
```

Comment:
Let's change the titles for the menu list to even just the names.

Reply:
Titles have been changed